src/openvic-dataloader/v2script/AbstractSyntaxTree.cpp

```diff
@@ -14,6 +14,22 @@
 
 using namespace ovdl::v2script::ast;
 
+static void _handle_string_characters(std::string& string, bool allow_newline) {
+	size_t position = 0;
```

Comment:
What does `position` do in this function?

Reply:
For now its doing nothing, but it'll be necessary to enable transitioning from win1252 to utf-8 characters if we actually do that.